pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -8738,6 +8743,71 @@
+  middle_high_page_cs_card_description_3: "Introduce students to software engineering and object-oriented design while they learn the Java programming language in this free curriculum for AP® Computer Science A (AP® CSA)."
+  middle_high_page_cs_link_1: "Explore CS Dicoveries"
+  middle_high_page_cs_link_2: "Explore CS Principles"
+  middle_high_page_cs_link_3: "Explore CSA"
+
+  elemeentary_page_main_title: "Learning for Ages 5 to 11"
+  elementary_page_main_description: "Explore the world of computer science with engaging videos, self-paced tutorials, programming activities, and more!"
+  elementary_page_own_pace_title: "Continue Learning at Your Own Pace"
+  elementary_page_own_pace_description: "Create drawings, games, and more in these two excellent options for students getting started on their own learning the basics of computer science."
+  elementary_page_own_pace_card_title_1: "Pre-Reader Express Course"
+  elementary_page_own_pace_card_title_2: "Express Course"
+  elementary_page_own_pace_card_description_1: "Learn computer science at your own pace! Learn to create computer programs, develop problem-solving skills, and work through fun challenges! Make games and creative projects to share with friends, family, and teachers."
+  elementary_page_own_pace_card_description_2: "Learn computer science at your own pace! Learn to create computer programs, develop problem-solving skills, and work through fun challenges! Make games and creative projects to share with friends, family, and teachers."
+  elementary_page_own_pace_link_1: "Start Pre-Reader Course"
+  elementary_page_own_pace_link_2: "Start Express Course"
+  elementary_page_make_your_own_games_title: "Make Your Own Animations and Games"
+  elementary_page_make_your_own_games_card_title_1: "Sprite Lab"
+  elementary_page_make_your_own_games_card_description_1: "Time to get creative! Use block-based coding to make something silly, sweet, or outer-spacey in Sprite Lab. The only limit is your imagination!"
+  elementary_page_make_your_own_games_link: "Try Sprite Lab"
+  elementary_page_computer_science_title: "Computer Science in the Classroom"
+  elementary_page_computer_science_description: "These courses should be led by a teacher! If you'd like to see them taught at your school, ask a parent or guardian to work with a teacher to bring them to your class."
+  elementary_page_computer_science_card_title_1: "Computer Science Fundamentals"
+  elementary_page_computer_science_card_title_2: "Computer Science Connections"
+  elementary_page_computer_science_description_1: "Free set of elementary curricula that introduces students to the foundational concepts of computer science and challenges them to explore how computing and technology can impact the world."
+  elementary_page_computer_science_description_2: "Make connections between computer science (CS) and other subjects like math, language arts, science, and social studies."
+  elementary_page_computer_science_link: "Explore curricula"
```

Comment:
Nit: can you add this in the [same section here](https://github.com/code-dot-org/code-dot-org/blob/b38b7cd849236dda00ed61ac907a348353696cb6/i18n/locales/source/hourofcode/en.yml#L211) so it's more reusable?